8bitzz/COSC2101
# Feat/user order

### Change-log
- Introduce `Order History` page to display all users' orders
- Install `swr` to handle fetched order data
- Add `UserDropdown` on navigation bar

### How-to-test
- Checkout this brach and run server on both frontend backend (`npm install` on frontend to install `swr`)
- Login with your account or using test account`madmax@example.com/A12345678`
- Verify that users can see an icon on navigation bar
- Select `My Order` option will navigate users to `Order History` page
- Verify that users see message **You haven't placed any orders yet** with button `Checkout` if they haven't placed any orders

### Known-issue
- Cart number on navigation bar have not be implemented. Should be fixed in upcoming PR 

### UI

<img width="1675" alt="Screen Shot 2021-08-24 at 14 10 39" src="https://user-images.githubusercontent.com/23531403/130578370-08865c38-3aa4-4933-90f8-024193ad7c7c.png">
